lib/editing/editor_codemirror.dart

```diff
@@ -181,6 +198,18 @@ class _CodeMirrorEditor extends Editor {
 
   void execCommand(String name) => cm.execCommand(name);
 
+  void showCompletions({bool autoInvoked: false, bool onlyShowFixes: false}) {
+    if (autoInvoked)
```

Comment:
Can you add braces (`{`) here and below? The dart style guide requires them except for single-line if statements.